staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fieldmanager.go

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
It's hard to tell what changes were made, if any, to the methods in this file, but it seems ok on surface

Reply:
Yeah, there are no changes beside `fieldmanager.` -> `internal.` AFAIR